js/repeater.js

```diff
@@ -546,9 +546,21 @@
 		runRenderer: function(container, renderer, data, callback){
 			var self = this;
 			var skipNested = false;
+			var index = 0;
 			var repeat, subset, i, l;
 
-			var loopSubset = function(index){
+			function loopSubset(){
```

Comment:
while I prefer declaring functions this way over creating a variable, in the past jsHint would yell about that... however, it does not seem to be complaining currently. Did we alter the jsHint settings? If so and we're cool with that, great! :+1: